pedigroup/model/group.py

```diff
@@ -0,0 +1,25 @@
+from model.exceptions.cannotBeRemovedException import CannotBeRemovedException
+
+class Group:
+
+    def __init__(self, name):
+        self.name = name
+        self.users = set()
+        self.orders = list()
+
+    def addUser(self, user):
+        self.users.add(user)
+
+    def removeUser(self, user):
+        if not self.users.__contains__(user):
+            raise CannotBeRemovedException("No se puede remover un usuario el cual no pertenece al grupo")
+        self.users.remove(user)
```

Comment:
Por temas de legibilidad, creo que es mas conveniente utilizar esta sintaxis de python:
```suggestion
    def removeUser(self, user):
        if user not in self.users:
            raise CannotBeRemovedException("No se puede remover un usuario el cual no pertenece al grupo")
        self.users.remove(user)
```